src/v2/Apps/Partner/partnerRoutes.tsx

```diff
@@ -81,10 +81,20 @@ export const partnerRoutes: RouteConfig[] = [
         prepare: () => {
           ShowsRoute.preload()
         },
+        prepareVariables: ({ partnerId }) => {
+          return {
+            partnerId,
+            featuredShowStatus: null,
```

Comment:
status filter in Metaphysics has a [default value](https://github.com/artsy/metaphysics/blob/85123061a8e4ea3ed8a9f40b6bb4cacff0287e5c/src/schema/v2/partner.ts#L539-L542) and to get all shows I have to sent null as the status argument value. But if I just write:

`showsConnection(status: null, first: 1, sort: FEATURED_DESC_END_AT_DESC, isDisplayable: true)`

relay removes status: null from the query, so I added status as a variable and it's work.
Maybe it make sence to add `ALL` option with null value to the `EventStatus` [here](https://github.com/artsy/metaphysics/blob/94801da507df1c65e39d1c8584ff6510ae218e05/src/schema/v2/input_fields/event_status.ts#L3)?

Reply:
Ah, very weird — yeah I think adding `ALL` would make sense